flomo/tracker.py

```diff
@@ -47,3 +49,18 @@ def get_sessions(self):
     def get_session(self, session_id: float):
         self.cursor.execute("SELECT * FROM sessions WHERE id = ?", (session_id,))
         return self.cursor.fetchone()
+
+
+def show_sessions():
+    db = Tracker()
+    sessions = db.get_sessions()
+    db.conn.close()
+
+    print(
+        tabulate.tabulate(
+            pandas.DataFrame(sessions),  # type: ignore
+            headers=["ID", "Tag", "Name", "Start Time", "End Time", "Total Time"],
```

Comment:
new headers and data:
```
|Session Date | Tag | Name | Total Time |
```

- Include time and date in session date.
- colorise tag based on `config.json`

Reply:
sure, also, do you want me to not store the `start_time` and `end_time` at all, or store em just dont show?

Reply:
> sure, also, do you want me to not store the `start_time` and `end_time` at all, or store em just dont show?

i think you need for the calculations and stuff, right? so just **don't show dem**